run_process_zephir_incremental.sh

```diff
@@ -165,10 +167,10 @@ fi
 # This should have already been copied to the archive/catalog
 rm ${BASENAME}.json.gz
 
-echo "DONE `date`" >> $REPORT_FILE
-cat $REPORT_FILE | mailx -s"$SCRIPTNAME report: $TODAY" $EMAIL
+echo "DONE `date`"
+# TODO run cleanup_directory.sh?
```

Comment:
Right now this leaves a bunch of stuff in the working directory. I figure once this has been running for a little while in kubernetes we can see what we want to archive vs. delete.